### ISSUE TITLE
ERROR: Cannot decode JSON from PhantomJS runner

### ISSUE THREAD

**Issue by antono** [guard/guard-jasmine#25]
```
ERROR: Cannot decode JSON from PhantomJS runner: 705: unexpected token at '2012-01-11T23:25:51 [DEBUG] Connected to accessibility bus at:  "unix:abstract=/tmp/dbus-kci3ZPUepy,guid=0aa6231e949e1034b6880b7300002fe9" 
2012-01-11T23:25:51 [DEBUG] Registered DEC:  true 
{
  "passed": false,
  "stats": {
    "specs": 22,
    "failures": 1,
    "time": 0.687
  },
```

I supose stderr somehow appeared in json so it cannot be parsed properly.
